src/components/molecules/filter/StateDropdown.jsx

```diff
@@ -3,7 +3,7 @@ import { useLocation } from "../../../contexts/FilterContext"
 
 export default function StateDropdown(){
 
-    const { selectState } = useLocation()
+    const { selectState, state } = useLocation()
 
     function selected(e){
         e.preventDefault()
```

Comment:
You might not need `preventDefault` for the `onChange` event.

Reply:
gotcha